lms/djangoapps/certificates/views/support.py

```diff
@@ -80,7 +80,7 @@ def search_certificates(request):
         ]
 
     """
-    user_filter = urllib.unquote(urllib.quote_plus(request.GET.get("user", "")))
+    user_filter = urllib.quote_plus(request.GET.get("user", ""), safe="@.!#$%&'*+-/=?^_`{|}~")
```

Comment:
have you tried using `bleach`?

Reply:
bleach is not working for `+` sign. It converts it into `space`.

Reply:
I don't like hardcoding it the `safe` values here. Lets think of some other way. 

This code is basically converting ` ` spaces to `+` signs and unquoting string again.

Reply:
What about something like 
`bleach.clean(urllib.unquote(urllib.quote_plus(request.GET.get("user", ""))))`

Reply:
Done.